service/ca/pki.go

```diff
@@ -1,137 +0,0 @@
-package ca
```

Comment:
I tried to have smaller PRs for this but because of the restructuring it was not really possible to refactor the business logic separately. Thus we remove the old ca and crt magic here and add a `vaultpki` and `vaultcrt` resource to do the magic instead.